File: code/base/hello/objectivec/hello.m
#import <Foundation/Foundation.h>

int main(int argc, const char * argv[]) {
  @autoreleasepool {
    NSError *error;
    NSString *str = @"こんにちは、世界！\n";

    // TODO: "/dev/stdout"が標準出力とならないWindowsシステムでは動作しない
    [str writeToFile:@"/dev/stdout"
          atomically:NO
          encoding:NSUTF8StringEncoding
             error:&error];
  }
  return 0;
}

File: code/type/number/intger/cplusplus/hello.cpp
#include <iostream>
#include <cstdint>

int main(int argc, char *argv[]) {
  int a = 1;
  std::cout << u8"こんにちは、世界！" << std::endl;
  std::cout << a << std::endl;
  std::cout << u8"こんにちは、世界！" << std::endl;
  return 0;
}
